File: 3195-separate-black-and-white-balls/3195-separate-black-and-white-balls.java
class Solution {
    public long minimumSteps(String s) {
        int black = 0;
        long swap = 0;
        // convert string into char of array
        for(char c : s.toCharArray()){
            if(c == '1'){
                black++;
            } else {
                swap += black;
            }
        }
        return swap;
    }
}

File: 3195-separate-black-and-white-balls/3195-separate-black-and-white-balls.py
class Solution(object):
    def minimumSteps(self, s):
        """
        :type s: str
        :rtype: int
        """
        black = 0
        swap = 0
        for i in s:
            if i == '1':
                black += 1
            else:
                swap += black
        return swap
        